c-pallets/file-bank/src/lib.rs

```diff
@@ -424,12 +432,12 @@ pub mod pallet {
 		/// - `file_hash`: The beneficiary related to signer account.
 		/// - `file_size`: File size calculated by consensus.
 		/// - `slice_info`: List of file slice information.
-		/// - `user`: The first user to upload files.
+		/// - `user`: The first user to upload files. // Does it mean origin? I don't see the parameter
```

Comment:
The origin of the extrinsic is consensus, so I need to know which user requests consensus.

Reply:
But I can't see `user` as a distpatchable parameter

Reply:
In fact, I don't use this parameter anymore. I'm sorry, but I didn't update the comment in time